internal/wrappers/jwt-helper.go

```diff
@@ -20,7 +20,7 @@ type JWTStruct struct {
 	jwt.Claims
 }
 
-var enabledEngines = []string{"sast", "sca", "api-security", "iac-security", "scs", "containers"}
+var enabledEngines = []string{"sast", "sca", "api-security", "iac-security", "scs", "containers", "enterprise-secrets"}
```

Comment:
I'm not sure I understand. enterprise-secrets is twoms? if so, what does the "scs" engine type stand for?

Reply:
Enterprise-secrets is a sublicense of SCS meaning that you can only have Enterprise-secrets license if you already have SCS license